packages/vue-test-app/index.html

```diff
@@ -6,12 +6,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>VueTestApp</title>
+    <base href="/" />
     <link rel="icon" type="image/svg+xml" href="/vue.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vue Test App</title>
   </head>
-  <body>
+  <body class="theme-classic-dark">
```

Comment:
Why is this class necessary?

Reply:
When the themeSwitcher package is utilized within the echarts example, the addition of this class becomes redundant.